libibverbs/verbs.h

```diff
@@ -1445,16 +1445,7 @@ struct ibv_wc_tm_info {
 };
 
 struct ibv_cq_ex {
-	struct ibv_context     *context;
-	struct ibv_comp_channel *channel;
-	void		       *cq_context;
-	uint32_t		handle;
-	int			cqe;
-
-	pthread_mutex_t		mutex;
-	pthread_cond_t		cond;
-	uint32_t		comp_events_completed;
-	uint32_t		async_events_completed;
```

Comment:
We should prevent changing a structure that is exposed from verbs.h , there might be some application that was written with direct usage of one on applicable/exposed fields (e.g. channel, cq_context, etc.) that post of such change might fail to compile.

ibv_qp_ex was introduced from day one with qp_base comparing this one.

Reply:
You're definitely right, missed that.